examples/units/basic_units.py

```diff
@@ -25,13 +25,13 @@ def __get__(self, obj, objtype=None):
 
 
 class TaggedValueMeta(type):
-    def __init__(cls, name, bases, dict):
```

Comment:
It may be that the object is a class/type here, but it's still an instance at this point, so I'm not sure calling it something other than `self` is warranted.